common/src/service/HttpClientHelper.cs

```diff
@@ -871,8 +871,9 @@ internal static HttpClient CreateDefaultClient(IWebProxy webProxy, Uri baseUri,
         internal static HttpMessageHandler CreateDefaultHttpMessageHandler(IWebProxy webProxy, Uri baseUri, int connectionLeaseTimeoutMilliseconds)
         {
             var httpMessageHandler = new HttpClientHandler();
-            httpMessageHandler.SslProtocols = TlsVersions.Instance.Preferred;
-            httpMessageHandler.CheckCertificateRevocationList = TlsVersions.Instance.CertificateRevocationCheck;
+            var options = new IotHubServiceClientOptions();
+            httpMessageHandler.SslProtocols = options.SslProtocols;
```

Comment:
Need to pipe in the IotHubServiceClientOptions that the user passed in the service client ctor.